packages/blade/src/components/Form/Selector/types.ts

```diff
@@ -1,11 +1,49 @@
+import type React from 'react';
 import type { TestID } from '~src/_helpers/types';
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
-export type SelectorLabelProps = {
+type SelectorLabelProps = {
   children: React.ReactNode;
   /**
    * Pass only on react-native
    */
   inputProps: any; // Partial<Extract<InputProps, { onPress: any }>>;
   componentName?: string;
+  onMouseDown?: React.MouseEventHandler<HTMLLabelElement>;
+  onMouseUp?: React.MouseEventHandler<HTMLLabelElement>;
+  onMouseOut?: React.MouseEventHandler<HTMLLabelElement>;
+  onKeyDown?: React.KeyboardEventHandler<HTMLLabelElement>;
+  onKeyUp?: React.KeyboardEventHandler<HTMLLabelElement>;
+  onTouchStart?: React.TouchEventHandler<HTMLLabelElement>;
+  onTouchEnd?: React.TouchEventHandler<HTMLLabelElement>;
+  style?: React.CSSProperties;
 } & TestID;
+
+type SelectorInputProps = HoverProps & {
+  id?: string;
+  inputProps: any;
+  tabIndex?: number;
+  accessibilityLabel?: string;
+};
+
+type SelectorInputHoverTokens = {
+  default: {
+    background: {
+      checked: string;
+      unchecked: string;
```

Comment:
Why are these strings? Shouldn't they be a type of color token?

Reply:
Changed.